package.json

```diff
@@ -12,7 +12,7 @@
     "phoenix": "npx @s-ui/mono phoenix",
     "phoenix:ci": "sui-mono phoenix --ci --no-root",
     "co": "sui-mono commit",
-    "lint": "sui-lint js && sui-lint sass",
+    "lint": "sui-lint js",
```

Comment:
There's a problem with "sui-lint scss" but I think it's worth closing this PR and solving it later